.gitignore

```diff
@@ -10,7 +10,9 @@ docs/schema.json
 node_modules
 .idea
 .env
-images
+image/*
+videos/*
+public/markdown/*
```

Comment:
Why was this included?

Reply:
In the reference PR, Eshaan told to ignore this directory, but you say, should I ignore or not?

Reply:
You are moving image files that are referenced in the documentation markdown files and then ignoring them so that they don't get uploaded to the repository. You can see that they will be ignored by the gray icons

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/115bc5ec-fc25-4fdd-b58e-f53eb5bdb139)